gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -133,6 +133,9 @@ private static void configureErrorProneOptions(
                 "CanIgnoreReturnValueSuggester",
                 "InlineMeSuggester",
                 "PreferImmutableStreamExCollections",
+                // StringCaseLocaleUsage duplicates our existing DefaultLocale check which is already
+                // enforced in some places.
+                "StringCaseLocaleUsage",
                 "UnnecessaryTestMethodPrefix",
```

Comment:
Getting failures on this, which is unexpected -- `UnnecessaryTestMethodPrefix` is not enabled by default, so I'll remove it from the list.

```suggestion
```